handlers/services.go

```diff
@@ -95,19 +95,18 @@ func ServiceDetails(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var istioConfigValidations = models.IstioValidations{}
+	var errValidations error
 
 	wg := sync.WaitGroup{}
 	if includeValidations {
 		wg.Add(1)
-		go func(istioConfigValidations *models.IstioValidations, err *error) {
+		go func(istioConfigValidations *models.IstioValidations) {
 			defer wg.Done()
 			istioConfigValidationResults, errValidations := business.Validations.GetValidations(namespace, service)
```

Comment:
This errValidations is not necessarily the pointer to the above errValidations as you use ``:=``. So inside the goroutine we have a different struct. You need to either pass it to the go func in the parameters or not use ``:=``

Reply:
oops you're right